### PR TITLE
[Scripts] Edit.js - Add color bars below XYZ inputs to indicate world axis

### PR DESCRIPTION
**Pull request by imgntn**
This PR adds a little bar of color below each XYZ input to show which axis it moves along.  Quick addition based on direct feedback from some alphas.  

To test: https://rawgit.com/imgntn/hifi/edit_better/examples/edit.js

capture

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: aa6aaf912e3d12b0e94dc3c106314361fc2cfe24 available at: https://gist.github.com/5943f942321e193cdccb<br/><br/>No installer was built for this commit

**Comment by hifi-gustavo** [comment]
Build console output for commit: c6fcaf158b221fd934fdc3861366783aa6114e4f available at: https://gist.github.com/5943f942321e193cdccb<br/><br/>No installer was built for this commit

File: examples/html/style.css [hunk 1]
```diff
@@ -342,3 +343,23 @@ input#property-name {
 	margin: 5px;
 	border: 1px solid white;
 }
+
+.prop-x{
```
Comment (huffman) [inline examples/html/style.css]: Can you put a space before `{`?

File: examples/html/style.css [hunk 2]
```diff
@@ -342,3 +343,23 @@ input#property-name {
...
+  color:red !important;
```
Comment (huffman) [inline examples/html/style.css]: indentation should be 4 spaces

**Comment by imgntn** [comment]
yep -- thanks @huffman

**Comment by hifi-gustavo** [comment]
Build console output for commit: 31cc451fd44e38e608b3377842a9d79ae94427e8 available at: https://gist.github.com/5943f942321e193cdccb<br/><br/>No installer was built for this commit

**Pull request merged**